modules/apps/object/object-service/src/main/java/com/liferay/object/internal/action/executor/AddObjectEntryObjectActionExecutorImpl.java

```diff
@@ -48,28 +69,205 @@ public void execute(
 			throw new UnsupportedOperationException();
 		}
 
-		_objectEntryLocalService.addObjectEntry(
-			userId,
-			GetterUtil.getLong(
-				parametersUnicodeProperties.get("groupId")),
-			GetterUtil.getLong(
-				parametersUnicodeProperties.get("objectDefinitionId")),
-			Collections.<String, Serializable>emptyMap(),
-			//(Map<String, Serializable>)parametersUnicodeProperties.get(
-			//	"values"),
-			new ServiceContext() {
-				{
-					setUserId(userId);
-				}
-			});
+		ObjectDefinition objectDefinition2 =
+			_objectDefinitionLocalService.fetchObjectDefinition(
+				GetterUtil.getLong(
+					parametersUnicodeProperties.get("objectDefinitionId")));
```

Comment:
I would have to rename in `public void execute(long companyId, UnicodeProperties parametersUnicodeProperties, JSONObject payloadJSONObject, long userId)` and it would look different from the others executors like `GroovyObjectActionExecutorImpl` and `WebhookObjectActionExecutorImpl`